modules/storages/spec/services/storages/project_storages/delete_service_spec.rb

```diff
@@ -145,4 +139,14 @@
 
     it_behaves_like("an event gun", OpenProject::Events::PROJECT_STORAGE_DESTROYED)
   end
+
+  private
+
+  def command_class_reference
+    if storage.provider_type_nextcloud?
+      Storages::Peripherals::StorageInteraction::Nextcloud::DeleteFolderCommand
+    else
+      Storages::Peripherals::StorageInteraction::OneDrive::DeleteFolderCommand
+    end
+  end
```

Comment:
Can't we use the `Registry` to solve this here?

Reply:
well, honestly I'm not sure. I just fixed a rubocop here.

this is the code:

```ruby
    let(:command_double) { class_double(command_class_reference, call: ServiceResult.success) }

    before do
      Storages::Peripherals::Registry
        .stub("#{storage.short_provider_type}.commands.delete_folder", command_double)
    end
```

before it was an unverified double on `:delete_folder_command`. I understand we need the double to expect calls on it? Is this solvable with the registry alone?

Reply:
Well, it works :D. It's just a bit weird, If I would inline it, then we would have this:

`Registry.stub("key", class_double(Registry.resolve("key"), call: Success))`

Is this weird? or fine? Asking for a friend.

Reply:
We only do `.call` over the command, right? So we could do:

```ruby
Registry.stub("key", ->(*) { Success })
```